mlir-tensorrt/python/bindings/Compiler/CompilerPyBind.cpp

```diff
@@ -240,6 +251,29 @@ PYBIND11_MODULE(_api, m) {
         return new PyCompilerClient(client);
       }));
 
+  py::class_<PyOptionsContext>(m, "OptionsContext", py::module_local())
+      .def(py::init<>([](PyCompilerClient &client,
+                         const std::string &optionsType,
+                         const std::vector<std::string> &args) {
+             std::vector<MlirStringRef> refs(args.size());
+             for (unsigned i = 0; i < args.size(); i++)
+               refs[i] = mlirStringRefCreate(args[i].data(), args[i].size());
+
+             MTRT_OptionsContext options;
+             MTRT_Status s = mtrtOptionsContextCreateFromArgs(
+                 client, &options, optionsType.c_str(), refs.data(),
+                 refs.size());
+             THROW_IF_MTRT_ERROR(s);
+             return new PyOptionsContext(options);
+           }),
+           py::arg("client"), py::arg("options_type"), py::arg("args"))
+
+      .def("print", [](PyOptionsContext &self) {
+        // TODO: Should this be exposed via __str__ instead?
```

Comment:
Might be cleaner to expose this as a `__str__` method, but this is primarily exposed for testing purposes, so not sure if it matters too much.

Reply:
Commented above on methodology the CAPI declaration